tools/wpt/expectation.json

```diff
@@ -3286,15 +3286,11 @@
       "scripting": {
         "reporterror.any.html": [
           "self.reportError(1)",
-          "self.reportError(TypeError)",
-          "self.reportError(undefined)",
-          "self.reportError() doesn't invoke getters"
+          "self.reportError(undefined)"
```

Comment:
https://github.com/web-platform-tests/wpt/blob/66d28833f7b932a1575531cb4b035a18047f8ad2/html/webappapis/scripting/reporterror.any.js#L12-L14

We don't provide location info for thrown values which aren't `Error` objects. Browsers usually fall back to the throw site in these situations, or in this case the `reportError()` call site. That's what the tests are expecting us to do. Of course that's trivial for browsers which implement `reportError()` as a direct native binding, since `v8::Exception::create_message(scope, exception)` would "just work". But for us we need to manually analyse the stack trace within `reportError()`.

Reply:
> But for us we need to manually analyse the stack trace within `reportError()`.

Done https://github.com/denoland/deno/pull/13799/commits/73aa722a4771e99a1d8ef115f90cff60c2589021.